docs/api/smart-contracts/get-smart-contracts-status.example.json

```diff
@@ -0,0 +1,31 @@
+{
+  "SP3K8BC0PPEVCV7NZ6QSRWPQ2JE9E5B6N3PA0KBR9.swap-helper-bridged-v1-1": {
+    "found": true,
+    "status": {
```

Comment:
Maybe it would be better to call this result, since not all the properties are related to status